tests/phpunit/integration/Modules/Thank_With_Google/Web_TagTest.php

```diff
@@ -59,20 +60,31 @@ public function test_content_placeholder_inserted_on_cta_post_types() {
 
 		$this->assertStringContainsString( 'Thank with Google snippet added by Site Kit', $output );
 		$this->assertStringContainsString( '<button twg-button', $output );
+		$this->assertStringContainsString( '<div counter-button', $output );
 	}
 
-	public function test_content_placeholder_not_inserted_on_dynamic_cta_placement() {
+	public function test_content_cta_placeholder_not_inserted_on_dynamic_cta_placement() {
 		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_DYNAMIC_LOW );
 
 		$this->create_post_and_go_to_it();
 
 		$output = apply_filters( 'the_content', $this->dummy_content );
 
-		$this->assertStringNotContainsString( 'Thank with Google snippet added by Site Kit', $output );
 		$this->assertStringNotContainsString( '<button twg-button', $output );
 	}
 
-	public function test_content_placeholder_inserted_static_below_content() {
+	public function test_content_counter_placeholder_inserted_on_dynamic_cta_placement() {
+		$this->web_tag->set_cta_placement( Web_Tag::PLACEMENT_DYNAMIC_LOW );
+
+		$this->create_post_and_go_to_it();
+
+		$output = apply_filters( 'the_content', $this->dummy_content );
+
+		$this->assertStringContainsString( 'Thank with Google snippet added by Site Kit', $output );
+		$this->assertStringContainsString( '<div counter-button', $output );
```

Comment:
It would be good to use `assertStringEndsWith` here to better verify the logic being tested.